addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -139,9 +143,11 @@ DuplicatePageDialog.template = xml`
 </WebsiteDialog>
 `;
 DuplicatePageDialog.props = {
+    duplicate: {type: Function, optional: true},
     onDuplicate: {type: Function, optional: true},
```

Comment:
Not related to your PR but it's weird that this one is optional while it is not (code will crash without it).
I made a PR to fix that in 16: https://github.com/odoo/odoo/pull/147600